test/inductor/test_flex_decoding.py

```diff
@@ -964,6 +964,43 @@ def test_do_not_trigger_dynamic_shapes_on_empty_block_mask(self):
             else:
                 self.assertEqual(torch._dynamo.utils.counters["frames"]["ok"], 2)
 
+    @supported_platform
+    def test_non_divisible(self):
```

Comment:
If its possible to make a smaller test might be good

I think I was able to hit it on iteration 65 pretty reliably before the fix to flex_decode